warehouse/templates/pages/help.html

```diff
@@ -35,7 +35,7 @@
     </p>
     <p>
       {% trans %}
-        We recommend that all PyPI users set up <em>at least two</em> supported two factor authentication methods
+        We recommend that all PyPI users set up <em>at least two</em> supported two-factor authentication methods
```

Comment:
lol can we make this recommended automatically turn in to a required when we flip da switch.

Reply:
Ope. Missed context. Thank you